packages/example-app-base/index.html

```diff
@@ -46,9 +46,9 @@
       }
 
       if (useMinified) {
-        scripts.push('demo-app.min.js');
+        scripts.push('demo.min.js');
```

Comment:
Was there a reason for this change? It broke aka.ms/fabricdemo and individual packages' PR deploy.